torchvision/prototype/datasets/_builtin/gtsrb.py

```diff
@@ -74,14 +87,12 @@ def _prepare_sample(self, data: Tuple[Tuple[str, Any], Dict[str, Any]]) -> Dict[
         return {
             "path": path,
             "image": EncodedImage.from_file(buffer),
-            "label": Label(label, categories=self.categories),
+            "label": Label(label),
```

Comment:
Why remove this? If we have access to the human readable categories, we should also put them on the labels. This makes data exploration a lot easier. I agree, for GTSRB it is probably not needed since there are no human readable categories, but in that case we can also remove the entry from the info dictionary.